draft-ietf-moq-transport.md

```diff
@@ -1334,11 +1374,13 @@ value is 0, so if not specified, the peer MUST NOT send requests.
 The MAX_EARLY_SUBSCRIPTION_DATA parameter (Parameter Type 0x04) communicates the
 maximum amount of data a publisher can send on a subscription using a
 Publisher-Chosen Subscription ID ({{subscription-id}}) that has not been
-acknowledged on the control stream via PSID_ACK ({{message-psid-ack}}).  This
-limit includes the sum of all bytes sent on Subgroup streams including the
-SUBGROUP_HEADER.  Resetting a stream does not increase the limit.  The default
-is 0.  The limit does not apply to DATAGRAM and DATAGRAM_STATUS.  Publishers are
-not limited when using the Request Subscription ID in SUBGROUP_HEADER.
+acknowledged on the control stream via PSID_ACK ({{message-psid-ack}}).  It also
+limits the amount of data that can be published on a subscription initiated with
+the PUBLISH message before PUBLISH_OK.  This limit includes the sum of all bytes
+sent on Subgroup streams including the SUBGROUP_HEADER.  Resetting a stream does
+not increase the limit.  The default is 0.  The limit does not apply to DATAGRAM
+and DATAGRAM_STATUS.  Publishers are not limited when using the Request
+Subscription ID in SUBGROUP_HEADER.
```

Comment:
May be this parameter must come in a separate PR?

Reply:
It's in #5, remember the order of PRs here is 1 -> 5 -> 3 -> 2

Reply:
As Cullen said in the meeting, it makes sense to limit the amount of data that a publisher can send before PUBLISH_OK, because the publisher is unauthorized prior to that.